ethereum_bridge/src/storage/eth_bridge_queries.rs

```diff
@@ -500,6 +500,20 @@ pub struct EthAssetMint {
     pub erc20_amount: token::Amount,
 }
 
+impl EthAssetMint {
+    /// Check if NUTs should be minted.
+    #[inline]
+    pub fn mint_nuts(&self) -> bool {
```

Comment:
nit: from these fn names starting with a verb it sounds like they perform minting - perhaps better `should_mint_x`

Reply:
sure, I'll make sure to change this after https://github.com/anoma/namada/pull/1755 merges